samples/DotNet/Microsoft.ServiceBus.Messaging/RoleBasedAccessControl/Program.cs

```diff
@@ -253,9 +177,107 @@ void InitializeReceiver()
                             Console.ResetColor();
                         }
                     }
-                    await message.CompleteAsync();
+                    await this.receiveClient.CompleteAsync(message.SystemProperties.LockToken);
                 },
-                new OnMessageOptions { AutoComplete = false, MaxConcurrentCalls = 1 });
+                new MessageHandlerOptions((eventargs) => { return Task.CompletedTask; })
+                {
+                    AutoComplete = false,
+                    MaxConcurrentCalls = 1
+                });
+        }
+
+        async Task UserInteractiveLoginScenario()
+        {
+            var qClient = QueueClient.CreateWithAzureActiveDirectory(
```

Comment:
>QueueClient.CreateWithAzureActiveDirectory [](start = 26, length = 42)

Not sure which Servicebus SDK this is, but with the CoreCLR SDK (from the azure-sdk-for-net repo) this signature shouldn't exist